README.md

```diff
@@ -26,7 +27,9 @@ const pluginOptions = {
 };
 const plugin = oniyiHttpPluginCredentials(pluginOptions);
 
-httpClient.use(plugin);
+const httpClient = OniyiHttpClient
+  .create(httpClientParams) // create custom http client with defined phase lists
+  .use(plugin);             // mount a plugin
```

Comment:
you can add `options` here to map the plugin's phase name to one of the existing names in the httpClient's phaseList.

see [mapping code](https://github.com/benkroeger/oniyi-http-client/blob/master/lib/helpers/mount-plugin.js#L51)

```js
httpClient.use(plugin, { requestPhaseMap: { credentials: 'initial' } });
```